tools/jetpack-live-branches/jetpack-live-branches.user.js

```diff
@@ -461,7 +472,38 @@
 		 */
 		function updateLink() {
 			const $link = $( '#jetpack-beta-branch-link' );
-			const [ url, query ] = getLink();
+			const [ url, query ] = getLink( host );
+
+			if ( url.match( /[?&]branch(es\.[^&=]*)?=/ ) ) {
+				if (
+					query.includes( 'jpcrm-populate-crm-data' ) &&
+					! url.match( /[?&]branches\.zero-bs-crm/ )
+				) {
+					// /jpcrm-populate-crm-data/
+					$link
+						.attr( 'href', null )
+						.text( 'Select the Jetpack CRM plugin in order to populate with CRM data' );
+				} else if (
+					query.includes( 'jpcrm-populate-woo-data' ) &&
+					! query.includes( 'woocommerce' )
+				) {
+					$link
+						.attr( 'href', null )
+						.text( 'Select the WooCommerce plugin in order to populate with CRM Woo data' );
+				} else {
+					$link.attr( 'href', url ).text( url );
+				}
+			} else {
+				$link.attr( 'href', null ).text( 'Select at least one plugin to test' );
+			}
+		}
+
+		/**
+		 * Update the link.
+		 */
+		function updateLink2() {
```

Comment:
Why did you not just add a `host` parameter to the function instead of copy-pasting it?

Reply:
Ok, and a parameter to specify which link to update.

Reply:
Because of its temporary nature after switching the jurassic.ninja domain these changes are going to be reverted.